lib/widgets/app_bars/searchable_app_bar.dart

```diff
@@ -29,6 +30,7 @@ class SearchableAppBar extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final responsiveUtils = ResponsiveUtils();
```

Comment:
It's bad,`ResponseUtils` is already registered in getX. Why don't you call it out to use?

Reply:
okay my bad